src/searchForm/datePicker/datePickerOverlay/DatePickerOverlay.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import cc from 'classcat'
+import Datepicker, { DatePickerProps, DatePickerOrientation } from 'datePicker'
+import Divider from 'divider'
+import DatePickerTopBar from 'searchForm/datePicker/datePickerTopBar'
+
+export interface DatePickerOverlayProps extends DatePickerProps {
+  readonly title: string
+}
+
+const numberOfMonths = 1
+
+export const DatePickerOverlay = (props: DatePickerOverlayProps) => (
```

Comment:
This component looks A LOT like the fullscreen one. Can't we fuse both of them by having two sets of default props (vertical, number of months) and a className for css differencies?

Reply:
Since they're internal components, the switch would be done on the Search Form component level. We talked with @grosbouddha about it and we thought it's easier to understand to have 2 distincts components, but this is probably challengeable

Reply:
It might useful as we add more differences in the future (like the datepicker might need JS based positioning). For now, I agree, it looks a bit weird to have 2 widgets.